CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 3.0.0
```

Comment:
when did we start tracking versions in the samples repo? but would be a good pattern to follow 👍